litestar/security/jwt/token.py

```diff
@@ -194,12 +195,18 @@ def decode(
         ) as e:
             raise NotAuthorizedException("Invalid token") from e
 
-    def encode(self, secret: str, algorithm: str) -> str:
+    def encode(
+        self,
+        secret: AllowedPrivateKeys | str | bytes,
```

Comment:
`str | bytes` is fine but we don't want to expose the pyjwt internals (`AllowedPrivateKeys`) here